src/appleseed.cli/main.cpp

```diff
@@ -608,39 +608,17 @@ namespace
         }
 
         // Write the frame to disk.
+        LOG_INFO(g_logger, "writing frame to disk...");
```

Comment:
I would suggest moving this message into `write_main_image()`.

Reply:
This message was redundant because just after saving the image we log it again, this time with some useful information like filename and timings.

Reply:
Ok!

Reply:
In that case, let's remove it then?